src/components/Modal/Modal.utils.ts

```diff
@@ -1,8 +1,17 @@
 export const COMPONENT_KEY = {
   Modal: 'Modal',
   Header: 'ModalHeader',
+  HeaderV2: 'ModalHeaderV2',
   Body: 'ModalBody',
   Content: 'ModalContent',
+  ActionFooter: 'ModalActionFooter',
   Footer: 'ModalFooter',
   Overlay: 'ModalOverlay',
 }
+
+export const MODAL_STYLES = {
```

Comment:
since that represent a modal kind (base on the prop name) I would rename that constant to MODAL_KIND